docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -18,6 +18,60 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## User experience updates in Kubernetes Monitoring
```

Comment:
So we haven't been grouping these by area/product because the idea for the next iteration of the Cloud what's new is that they'd be sortable by date or any number of other tags. "Ungrouping these" would entail:

- Removing the current H2 heading and text
- Making all the H3s into H2s
- Updating the titles to make clear that they relate to K8s
- Adding the contributor and availability information to each of entry 

I don't want to create extra work for you, but I don't know how this will translate when we switch to a new system in November/December. So I'll leave it up to you how you want to proceed.

Reply:
Interesting. Within K8s Mon are various releases, plus the plugin version. I'll talk to dev about this today. If we ungroup things, the things still show up in the same GUI. Will that be confusing to users? Or tedious because of the repeated "Kubernetes Monitoring" in the heading? I'm happy to do whatever works.